build.gradle

```diff
@@ -79,24 +124,19 @@ artifactory {
         contextUrl = 'https://ci-artifactory.corda.r3cev.com/artifactory'
         repository {
             repoKey = 'corda-dev'
-            username = 'teamcity'
-            password = System.getenv('CORDA_ARTIFACTORY_PASSWORD')
+            username = System.getenv('CORDA_ARTIFACTORY_USERNAME') || System.getProperty('corda.artifactory.username')
+            password = System.getenv('CORDA_ARTIFACTORY_PASSWORD') || System.getProperty('corda.artifactory.password')
         }
 
         defaults {
-            // Publish utils does not have a publish block because it would be circular for it to apply it's own
-            // extensions to itself
-            if(project.name == 'publish-utils') {
-                publications('publishUtils')
-            // Root project applies the plugin (for this block) but does not need to be published
-            } else if(project != rootProject) {
-                publications(project.extensions.publish.name())
+            if (publishProjects.contains(project)) {
+                publications(project.name)
             }
         }
     }
 }
 
 wrapper {
-    gradleVersion = "4.4.1"
+    gradleVersion = "4.8"
```

Comment:
_No, **not this**!_ We want [Gradle 4.8.1](https://github.com/gradle/gradle/releases/tag/v4.8.1)!!!!!!! :smiling_imp:

Reply:
Hah, perfectly bad timing :D